ch2driver/pytpcc/runtime/loader.py

```diff
@@ -165,7 +180,15 @@ def loadWarehouse(self, w_id):
             for c_id in range(1, self.scaleParameters.customersPerDistrict+1):
                 badCredit = (c_id in selectedRows)
                 orderDate = self.computeRandomRangeDate(startOrderDate, endOrderDate)
-                c_tuples.append(self.generateCustomer(w_id, d_id, c_id, orderDate, badCredit, True))
+                c_tuple, ca_tuples, cp_tuples, cc_tuples = self.generateCustomer(w_id, d_id, c_id, orderDate, badCredit, True)
+                c_tuples.append(c_tuple)
+                if (self.genFlatSchema()):
+                    for ca in ca_tuples:
+                        c_addresses_tuples.append(ca)
+                    for cp in cp_tuples:
+                        c_phones_tuples.append(cp)
+                    for cc in cc_tuples:
+                        c_categories_tuples.append(cc)
```

Comment:
Again I think these for loops can each be replaced with `+=` statements:
```
c_addresses_tuples += ca_tuples
c_phones_tuples += cp_tuples
c_categories_tuples += cc_tuples
```
Technically I don't think we even need the if statement since `[ca,cp,cc]_tuples` will all be empty given how their respective generation methods are written

Reply:
Done. The if statement is not needed as you correctly point out, but I have retained it so it is obvious to the reader that this is for a flat schema.